typescript-timers/interval.ts

```diff
@@ -0,0 +1,17 @@
+const $message = document.querySelector('h1');
+
+function countDown(): void {
+  if (!$message) throw new Error('$message not found!');
+  if (!($message.textContent === null)) {
+    if ($message.textContent.match(/\d+/)) {
+      if ($message.textContent === '0') {
+        $message.textContent = '~Earth Beeeelooowww Us~';
+        clearInterval(theCountDown);
+      } else {
+        $message.textContent = String(Number($message.textContent) - 1);
+      }
```

Comment:
This is funky because you are using the `textContent` which you needed to convert to a number to subtract 1 and then convert back to a string to assign to the `textContent` property. Part of this will be fixed by using a variable.